test/Test/Pos/CryptoSpec.hs

```diff
@@ -362,3 +411,122 @@ encToPublicToEnc =
 skToSafeSigner :: Crypto.SecretKey -> Property
 skToSafeSigner =
     Crypto.safeToPublic . Crypto.fakeSigner .=. Crypto.toPublic
+
+verifyEncShareGoodData :: SharedSecrets -> Bool
+verifyEncShareGoodData SharedSecrets
+    { getSecretSharing = secShare
```

Comment:
This formatting and pattern matching by records doesn't look pretty... Why not use `RecordsWildCards` here instead? Also, seems like `SharedSecrets` fields are not named by the convention in out style-guide.